package.json

```diff
@@ -61,6 +63,7 @@
     "rollup-plugin-typescript2": "^0.26.0",
     "sapper": "^0.27.0",
     "svelte": "^3.0.0",
+    "svelte-preprocess": "^4.0.6",
     "svelte-ts-preprocess": "^1.1.3",
```

Comment:
I'm not sure there's any reason to have `svelte-ts-preprocess` if you have `svelte-preprocess`, which handles TypeScript

Reply:
Sure, my mistake
I will make a next PR with the removing of all unusable dependencies

Could you please create issue for it?

Reply:
https://github.com/Zimtir/SENT-template/pull/10